CoEpi/ui/alerts/AlertsViewModel.swift

```diff
@@ -1,4 +1,5 @@
 import Dip
+import UIKit
```

Comment:
For what is `UIKit` needed?

Reply:
My original approach used a translation function from UIKit. Swiftgen replaced that function, so I will remove the UIKit import.

I'll have instructions added to README soon.

Reply:
Fixed both issues.